### ISSUE TITLE
Branch destruction once feature is finished?

### ISSUE THREAD

**Issue by fincamd** [Arquisoft/viade_en3a#11]
Guys, do you think that we should remove branches when we finish working on them and everything is merged into parent branch?

Because I think that it is a complete mess... Even now we already have near 7 branches and we haven't even started yet. Later this is going to be a clown fiesta.

What do you think?

:D

**Comment by PabloFerMar** [comment]
I think the branches we have created now could be deleted once we are done, since they are just about especific parts of the documentation. 
Once we start coding, depends on how we create the branches. I think everything that is in master has to work, so we wouldn't delete the other branches where we work on it.
Also we could just have one branch for documentation, but we would have to deal with conflicts. 
We could ask @pglez82 next Thursday just in case.

**Comment by fincamd** [comment]
I agree with you on asking @pglez82 next day.

Also, a new idea came to my mind while reading your message. I think it would be a good idea to have a development branch just as an intermediate step in between any new feature branch and master. Just in case we mess up while creating pull requests or sth.

I think we were told something like this in previous SDI seminars. I remember Edward saying something about this.

Do you all agree??  :)

**Comment by pglez82** [comment]
Hi. Branches once they are merged with the base branch should be removed to keep everything tidy.

The idea of creating a development branch as a middle step is also good.

Pablo

**Comment by Drastur** [comment]
Yes, in my opinion they should be deleted when done to keep the repository clean. As Pablo says, we can discuss it again once we start coding.

About the development branch, I  think is a great idea! Totally down for it. :P

**Comment by fincamd** [comment]
Development branch created as an intermediate step in between any new feature in master.

Closing this issue
